R/ji_completion.R

```diff
@@ -22,3 +22,35 @@ print.emoji_completion <- function(x, ...){
   writeLines( paste(x, " : ", names(x)) )
   invisible(x)
 }
+
+emoji_completer <- function(env) {
+  if (completeme::inside_quotes(env) && grepl(":[^[:space:]]*:?$", env$token)) {
+    # The IDE does not tokenize the words like the console
+    token <- sub("[^:]*:", ":", env$token)
+
+    if (grepl("^:[^[:space:]]+:$", token)) {
+      emoji <- emo::ji(gsub(":", "", token))
+    } else {
+      res <- ji_completion(sub(":", "", env$token))
```

Comment:
@jimhester I think this `env$token` should be `token` otherwise completion does not work for strings like "  :cat" 
(with things before the `:`)